docs/cloud9.md

```diff
@@ -0,0 +1,25 @@
+## Description
+Cloud9 IDE is an online integrated development environment. It supports hundreds of programming languages.
```

Comment:
Suggest we change the description since this isn't a "sales forum" :grin: 
Addition of how it's installed etc should probably be described since there's very little about it.
A link to the location for a local install would be recommended.

If that's not available let's limit this to:
> Cloud9 IDE is an online integrated development environment.

Reply:
Changed to "Cloud9 SDK is an webservice IDE that makes it easy to manage you configuration files."